src/actions/getRoomInfo.js

```diff
@@ -0,0 +1,26 @@
+import { getRoomsPromise } from './helpers/getRoomsHelper';
+import { getPlayersPromise } from './helpers/getPlayersHelper'
+
+const getRoomInfo = () => {
+  return async function (dispatch) {
+    try {
+      const roomIds = await getRoomsPromise();
+      let rooms = roomIds.map((roomId) => {
```

Comment:
Do we need to create object of array here?

According to `getPlayersPromise` function, the function use only roomId. Maybe you can pass `roomIds` as an argument of `getPlayersPromise`.